src/SortableMenuManageExtension.php

```diff
@@ -54,7 +54,6 @@ public function updateCMSFields(FieldList $fields)
             if (!$sortableMenuTab instanceof TabSet) {
                 throw new SortableMenuException('Sortable Menu must be a "TabSet", not "'.get_class($sortableMenuTab).'"');
             }
-            $sortableMenuTab = $fields->findOrMakeTab('Root.SortableMenu', 'Menus');
```

Comment:
this is not needed because of the code above. Also, the name is wrong resulting in a duplicate tab.